README.md

```diff
@@ -0,0 +1,93 @@
+# Checkly Terraform provider
+
+[![CircleCI](https://circleci.com/gh/bitfield/terraform-provider-checkly/tree/master.svg?style=svg)](https://circleci.com/gh/bitfield/terraform-provider-checkly/tree/master)
+
+This Terraform provider enables users to manage [Checkly](https://checklyhq.com) resources like checks.
+
+## Using the provider
+
+1. Download the binary applicable for your platform from the [latest tagged release](https://github.com/bitfield/terraform-provider-checkly/releases). 
+Then copy the binary to your Terraform plugin folder, unzip it and rename it to just `terraform-provider-checkly`. Lastly, set the correct access rights.
+
+
+I you're on MacOS, just run the script `install.sh` script
```

Comment:
We could have this work for Linux and other platforms too, couldn't we, if the script selected the right binary for the kernel and architecture?

Reply:
Probably, for Linux anyway. For Windows I have no idea how that should work. Shall I add one just for Linux and let the Windows part as an exercise to the reader?

Reply:
Yes, I think MacOS and Linux are the low-hanging fruit; if we can make the install completely seamless for those users, that's a big win.

Reply:
Okido! Should be good in the last update.